### DISCUSSION TITLE
Support for attaching custom fixers that are defined separetly to the plugin

### DISCUSSION THREAD

**Discussion by johnmathews** [w0rp/ale#3841] [General]
It would be useful to be able to specify a directory outside the Plugin where ALE can look for and register custom fixers.

In order to register a new fixer it is necessary to edit files that are part of the plugin package. This makes it cumbersome to create custom fixers and register them in multiple environments.

**Comment by hsanson** [comment]
Is possible to define custom fixers on your vim configuration without modifying ALE itself:

> :h ale#fix#registry#Add()
